src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -332,78 +339,101 @@ export function openFacilityPageV2(page, uiSchema, schema) {
       const initialState = getState();
       const newAppointment = initialState.newAppointment;
       const typeOfCare = getTypeOfCare(newAppointment.data);
-      const typeOfCareId = typeOfCare.id;
-      const siteIds = selectSystemIds(initialState);
-      const useVSP = vaosVSPAppointmentNew(initialState);
-      let typeOfCareFacilities = newAppointment.facilities[typeOfCareId];
-      let siteId = null;
-      let facilityId = newAppointment.data.vaFacility;
-      let parentFacilities = newAppointment.parentFacilities;
+      const typeOfCareId = typeOfCare?.id;
+      if (typeOfCareId) {
```

Comment:
There aren't very many changes within this `openFacilityPageV2` action aside from adding this `if (typeOfCareId)` statement around most of it.